types/node-hrx/index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module "node-hrx" {
+  import { Readable } from "stream"
+
+  class File {
+    path: string
+    body: string
+    comment?: string
+    isDirectory(): this is Directory
```

Comment:
isn't this simpler to write it as `false` here and `true` below (and the opposite in `Directory`), as an item can never be both a File and a Directory